src/scene/graph-node.js

```diff
@@ -24,18 +24,41 @@ const target = new Vec3();
 const up = new Vec3();
 
 /**
- * Callback used by {@link GraphNode#find} and {@link GraphNode#findOne} to search through a graph
- * node and all of its descendants.
+ * Helper function that handles signature overloading to receive a test function.
+ *
+ * @param {Function|string} attr - Attribute or lambda.
+ * @param {*} [value] - Optional value in case of `attr` being a `string`
+ * @returns {Function} Test function that receives a GraphNode and returns a boolean.
```

Comment:
```suggestion
 * @param {FindNodeCallback|string} attr - Attribute or lambda.
 * @param {*} [value] - Optional value in case of `attr` being a `string`
 * @returns {FindNodeCallback} Test function that receives a GraphNode and returns a boolean.
```

Just realized that the callback type is already defined and we can just reuse it here to make it more explicit